### PR TITLE
Timeline

### LINKED ISSUE #6: Update Timetable element to show events
- [ ] Must show events per day
- [ ] Must be responsive
- [ ] Must initially show the current day

### PR DESCRIPTION
**Pull request by skamoen**
This PR adds a Timeline. 

Credits to @svenpopping but he's too lazy to PR.

Fixes #6

### CODE REVIEW

**Comment by skamoen** [comment]
It defaults on Sunday, I'd put this on the Friday instead

File: app/elements/lancie-timeline/lancie-timeline.html [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+                  </tr>
+                  <tr>
+                    <td colspan="3"></td>
+                    <td class="timeblock" colspan="4">
+                      <div class="time-header">11:00-15:00</div>
+                      <div>HearthStone Final</div>
+                    </td>
+                    <td colspan="1"></td>
+                  </tr>
+                  <!-- <tr>
```
Comment (skamoen) [inline app/elements/lancie-timeline/lancie-timeline.html]: Commented code can be removed

File: app/elements/lancie-timeline/lancie-timeline.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    
+    .timeblock .time-header {
+      font-size: 11px;
+      font-weight: bold;
+      color: #666;
+      line-height: 16px;
+    }
+    </style>
+
+    <iron-ajax auto url="{{json}}" handle-as="json" last-response="{{activities}}" debounce-duration="300"></iron-ajax>
```
Comment (skamoen) [inline app/elements/lancie-timeline/lancie-timeline.html]: Does this do anything?

**Comment by skamoen** [comment]
Quick and dirty, but it works so fine for now.

File: app/elements/lancie-timeline/lancie-timeline.html (hunk 2 above)
Comment (svenpopping) [inline app/elements/lancie-timeline/lancie-timeline.html]: No not anymore :)

**Pull request merged**